misc/python/materialize/checks/all_checks/cluster_unification.py

```diff
@@ -33,9 +33,12 @@ def manipulate(self) -> list[Testdrive]:
             # Create either a source or a view as first object in cluster
             Testdrive(
                 """
-                > CREATE SOURCE shared_cluster_storage_first_source
+                >[version<9200] CREATE SOURCE shared_cluster_storage_first_source
```

Comment:
I think we're already on v0.93.0-dev already, so would be better to go up one version here and below.